tests/data_platform/test_models_exports.py

```
@@ -0,0 +1,21 @@
+from __future__ import annotations
+
+import data_platform.models as models
```

Ruff (0.16.3)

[warning] Use `from data_platform import models` in lieu of alias

Replace with `from data_platform import models`

(PLR0402)